packages/gatsby/src/commands/develop.js

```diff
@@ -426,6 +426,7 @@ module.exports = async (program: any) => {
       printDeprecationWarnings()
       if (program.open) {
         require(`opn`)(urls.localUrlForBrowser)
+          .catch(err => console.log(`${chalk.yellow(`warn`)} browser not opened`))
```

Comment:
i'd change this to something more descriptive. This sounds like it just decided not to open the browser

Reply:
I've also thought about that, but couldn't find a good wording - maybe "No web browser found, not opening" ?